### ISSUE TITLE
jsonlogger warning

### ISSUE THREAD

**Issue by MarcusJellinghaus** [MarcusJellinghaus/mcp-code-checker#29]
There is this warning, eg when running pytest

```
../../../../../opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pythonjsonlogger/jsonlogger.py:11
  /opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pythonjsonlogger/jsonlogger.py:11: DeprecationWarning: pythonjsonlogger.jsonlogger has been moved to pythonjsonlogger.json
    warnings.warn(
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

What can be done?
